src/test/java/ovh/knowall/socialstats/api/twitter/ShowMeWhatYouGotApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package ovh.knowall.socialstats.api.twitter;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ShowMeWhatYouGotApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
```

Comment:
#1 
```
@ExtendWith(SpringExtension.class)
@ContextConfiguration(classes = SpringJUnitConfigTest.Config.class)
class ShowMeWhatYouGotApplicationTests {

@Autowired
private ApplicationContext applicationContext;

@Test
void givenAppContext_WhenInjected_ThenItShouldNotBeNull() {
    assertNotNull(applicationContext);
}}

```
What do you think about it? Is it worth it?

Reply:
Sure, we can add it... but maybe in an MR related to the Prototype feature?